migrations/2023/Version20230417220111.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230417220111 extends AbstractMigration
+{
+  public function getDescription(): string
+  {
+    return '';
+  }
+
+  public function up(Schema $schema): void
+  {
+    // this up() migration is auto-generated, please modify it to your needs
+    $this->addSql('CREATE TABLE FeatureFlag (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, value TINYINT(1) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+    $this->addSql('DROP TABLE feature_flag');
+    $this->addSql('ALTER TABLE studio_activity CHANGE type type ENUM(\'comment\', \'project\', \'user\')');
+    $this->addSql('ALTER TABLE studio_user CHANGE role role ENUM(\'admin\', \'member\'), CHANGE status status ENUM(\'active\', \'banned\', \'pending_request\')');
+    $this->addSql('ALTER TABLE user_like_similarity_relation CHANGE similarity similarity NUMERIC(4, 3) DEFAULT \'0\' NOT NULL');
+    $this->addSql('ALTER TABLE user_remix_similarity_relation CHANGE similarity similarity NUMERIC(4, 3) DEFAULT \'0\' NOT NULL');
```

Comment:
FYI: those lines are currently generated at all times :/
It's a bug in the migration diff command in combination with our Entites & DB. You can just delete them before committing the migration file. On the other hand this results in no issues... just leaves the file with unnecessary content